lib/screens/last_trip.ex

```diff
@@ -0,0 +1,41 @@
+defmodule Screens.LastTrip do
+  @moduledoc """
+  Supervisor and public interface for fetching information about the last trips
+  of the day (Last Train of the Day, LTOTD).
```

Comment:
Very nitpicky: This looks to be the only place where it's called `Last Train` instead of `Last Trip`. Trip seems a little better because this isn't mode specific.

Reply:
hmm i put these parenthetical in here so someone searching for "last **train**" or "LTOTD" would still find the relevant code. would you still like me to remove these?

Reply:
how about [this?](https://github.com/mbta/screens/compare/0c9ca14d32f20fa5ad47b1999608bc9af8b7404f..413ff59c88929bcf9465b04022b3bcb383538f94)

Reply:
Yeah, was actually mid-comment suggesting something like that. Nice!